Snippets/Snippets_5/Outbox/SqlServer/ConnectionStrings.xml

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <!--startcode OutboxSqlServerConnectionStrings-->
 <configuration>
+  <configSections>
+    <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core"/>
```

Comment:
Because in the sample we mention auditing, but without that section it doesn't work. So the sample was broken before.